docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+
+```xml
+ <method_I enabled="true">
+   <measurement_area id="1" start_frame="None" stop_frame="None"/>
+   <cut_by_circle enabled="true" radius="1.0" edges="10"/>
+ </method_I>
+```
+
+Possible parameters are:
+
+- For each `measurement_area`, several id numbers can be set in one inifile.
+  `start_frame` and `stop_frame` give the starting and ending frame for data analysis.
+  The default values of these two parameters are `None`.
+  If you plan to analysis the whole run from beginning to the end,
+  set both of `start_frame` and `stop_frame` as `None`;
+  If `start_frame = None` but `stop_frame` is not,
+  then analysis will be performed from beginning of the trajectory to the `stop_frame`.
+  If `start_frame` is not `None` but `stop_frame = None`,
+  it will analyze from the `start_frame` to the end of the movement.
+
+- `cut_by_circle` determines whether to cut each cell by circle or not.
+         Two options `radius` of the circle and the number of `edges` have
+         to be supplied for approximating the circle if `enabled` is *true*.
+
+The data are saved in the output folder: `/Fundamental_Diagram/IndividualFD/` in the output file `IFD_I_` with
+Voronoi density $$\rho_i$$, velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon of each pedestrian $$i$$.
+
+
+### Method J
+For definition see [Method J](jpsreport_method_J.html). Method J is used to analyze the steady state.
+
+```xml
+ <method_J enabled="true">
+   <measurement_area id="1" start_frame="None" stop_frame="None"
+         get_individual_FD="false"/>
+   <one_dimensional enabled="false"/>
+   <cut_by_circle enabled="false" radius="1.0" edges="10"/>
+   <profiles enabled="false" grid_size_x="0.20" grid_size_y="0.20"/>
+ </method_J>
+```
+
+Possible parameters are:
+
+- For each `measurement_area`, several id numbers can be set in one inifile.
+    - `start_frame` and `stop_frame` give the starting and ending frame for data analysis. The default values of these two parameters are `None`. If you plan to analysis the whole run from beginning to the end, set both of `start_frame` and `stop_frame` as `None`; If `start_frame =None` but `stop_frame` is not, then analysis will be performed from beginning of the trajectory to the `stop_frame`. If `start_frame` is not `None` but `stop_frame = None`, it will analyze from the `start_frame` to the end of the movement.
+
+    - `get_individual_FD` determines whether or not to output the data for individual fundamental diagram in the given measurement area, which is based on the Voronoi density $$\rho_i$$,  velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon of each pedestrian $$i$$ in a given measurement area but not mean value over space. If true, the related data will be written in the folder `./Output/Fundamental_Diagram/IndividualFD/` in the output file `IFD_J_`.
+
+- `one_dimensional` should be used when pedestrians move on a line
```

Comment:
This could also be a separate tutorial (see above).

Reply:
I suggest having a separate PR for tutorials. 
What do you think?

Reply:
A good idea. Can you open a new issue after this PR?

Reply:
Thanks! #633